src/main/java/org/sonarsource/scanner/cli/SystemInfo.java

```diff
@@ -19,7 +19,19 @@
  */
 package org.sonarsource.scanner.cli;
 
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
 class SystemInfo {
+  private static final Set<String> SENSITIVE_JVM_ARGUMENTS = Stream.of(
+      "-Dsonar.login",
+      "-Dsonar.password",
+      "-Dsonar.token")
+    .collect(Collectors.toCollection(HashSet::new));
+  private static final Pattern patternArgumentSeparator = Pattern.compile("\\s+");
```

Comment:
not following naming convention for static final fields :slightly_smiling_face:

Reply:
Arf, I keep forgetting this 🙄 thxx